fbw-a380x/src/systems/instruments/src/MFD/pages/common/DataEntryFormats.tsx

```diff
@@ -200,6 +200,52 @@ export class AltitudeFormat implements DataEntryFormat<number> {
   }
 }
 
+export class RadioMinimumFormat implements DataEntryFormat<string | number> {
+  public placeholder = '-----';
+
+  public maxDigits = 5;
+
+  private minValue = 1;
+
+  private maxValue = maxCertifiedAlt;
+
+  constructor(
+    minValue: Subscribable<number> = Subject.create(1),
+    maxValue: Subscribable<number> = Subject.create(maxCertifiedAlt),
+  ) {
+    minValue.sub((val) => (this.minValue = val), true);
+    maxValue.sub((val) => (this.maxValue = val), true);
+  }
+
+  public format(value: string | number) {
+    if (value === null || value === undefined) {
+      return [this.placeholder, null, 'FT'] as FieldFormatTuple;
+    } else if (value === 'NONE' || value === 'NO' || value === 'NO DH' || value === 'NODH') {
```

Comment:
If the `value` type can only have those three string values or a number value, I would make the formatter type be:

```
type RadioMinimumEntry = 'NONE' | 'NO' | 'NO DH' | 'NODH' | number;
```

Reply:
Okay, done!